### DISCUSSION TITLE
Information about jointjs v4

### DISCUSSION THREAD

**Discussion by frnora** [clientIO/joint#2476] [Q&A] [answered]
Hi,

Could you share some information about v4 of jointjs? (or is there a way to track the progress?)

Do you have any ETA?

Looking at e.g. #2386, #2414, #2399 I see work's being done removing the dependencies. Will v4 have no dependencies whatsoever?

Will there be a migration guide from v3 to v4?

Thank you.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment, marked as answer]
It will be released in January.

An alpha version is already available.
```
npm install jointjs@alpha
```
```
yarn add jointjs@alpha
```

---

> Will v4 have no dependencies whatsoever?

Yes,`jQuery`, `Lodash` and `Backbone` have dropped.

The `DirectedGraph` has moved to a new package `@joint/layout-directed-graph`. The `dagre` and `graphlib` dependencies remain.

> Will there be a migration guide from v3 to v4?

Yes.
